### ISSUE TITLE
The installation is not correct, check your php.ini file.

### ISSUE THREAD

**Issue by mubeenanees** [opensourcepos/opensourcepos#1856]
PHP needs to have php-gd, php-bcmath, php-intl, php-sockets, php-mcrypt and php-curl installed and enabled... as said in the README.md. 
How do i install those things which you asking, I am new in PHP.
Please help
thanks
![ini](https://user-images.githubusercontent.com/29973402/36526303-90baa2f0-17cf-11e8-8a9b-ca4863d02810.PNG)

**Comment by jekkos** [comment]
Just do the cloud install in case you don't know how to configure PHP

**Comment by mckaygerhard** [comment]
umm @jekkos te only cloud install are paid, so i dont know, but its not a "user" option..  and some of that modules are just for send trivial information

**Comment by lennerv** [comment]
Hi there,

This is the first time I use this site. I recently installed OSPOS on a hosting paid which uses cpanel to modify PHP extensions. I checked that all extension were enable php-gd, php-bcmath, php-intl, php-sockets, php-mcrypt except by this php-curl which the vendor says it is installed by default.

I fixed the issue by changing the PHP version for 5.6 and make sure all extension mentioned are enable if not, OSPOS is not going to work.


![phpextension](https://user-images.githubusercontent.com/37355488/37380901-3e3297e6-2700-11e8-8867-cd081862e8df.PNG)
